### DISCUSSION TITLE
How can I convert every image file in my pictures folder?

### DISCUSSION THREAD

**Discussion by LAPIII** [ImageMagick/ImageMagick#5515] [Help] [answered]
I'm using Windows and I went to convert every image file to the JPEG format.

**Comment by snibgo** [comment]
Something like this:
```
magick mogrify -format JPG *.png
```

**Reply by LAPIII** [reply]
Is this correct: `"C:\Users\LPIII\Pictures" magick mogrify -format JPG *.png`?

**Comment by snibgo** [comment]
What is "C:\Users\LPIII\Pictures"? Is that your console prompt?

The command I showed will create JPG versions of all PNG files in the current directory.

**Comment by LAPIII** [comment]
`"C:\Users\LPIII\Pictures"` is my pictures folder.

**Comment by snibgo** [comment, marked as answer]
So, `cd` to whatever directory you want, then type or paste the command I showed.
